docs/src/components/landing-page/TwitterShowcaseCards.vue

```diff
@@ -142,16 +161,23 @@ export default defineComponent({
 .carousel-grid {
   display: grid;
   grid-template-columns: repeat(3, 1fr);
-  @media (max-width: $breakpoint-xs-max) {
-    grid-template-columns: repeat(1, 1fr);
+
+  @media screen and (max-width: $breakpoint-sm-max) {
```

Comment:
Latest Vue version allow v-bind into stylesheets
You could return the number of elements per chunk into setup method and bind it to the number of repetition into grid-template-columns, thus avoiding media queries and make the rule controlled by the JS part